CrowdStrike/crowdstrike_falcon/ingest/parser.yml

```diff
@@ -141,6 +174,114 @@ stages:
+          "0x00000040": "Duplicated EPA protocol fields"
+          "0x00000080": "Duplicated AV Flags protocol field"
+          "0x00000100": "Metasploit (new signature)"
+        mapping:
+          parsed_event.message.event.ProtocolAnomalyClassification: crowdstrike.protocol_anomaly_classification
+
+      - translate:
+        dictionary:
+          "1": "Domain replication"
+          "2": "SPN modification"
+          "3": "Replication server registration"
+          "4": "Scheduled task creation"
+          "5": "Remote service configuration"
+          "6": "Net session enumeration"
+          "7": "User management (SAMR)"
+          "8": "Remote SCM activation (DCOM)"
+          "9": "Remote task management"
+          "10": "Print system asynchronous remote protocol"
+          "11": "Netlogon server authenticate"
+        mapping:
+          parsed_event.message.event.RpcOpClassification: crowdstrike.rpc_op_classification
+
+      - translate:
+        dictionary:
+          "0": "SPN removal"
+          "1": "Machine sAMAccountName changed"
+        mapping:
+          parsed_event.message.event.SuspiciousMachineAccountAlterationType: crowdstrike.suspicious_machine_account_alteration_type
+
+      - set:
+          "@timestamp": "{{ parse_event_context_timestamp.datetime }}"
+          agent.id: "{{ parsed_event.message.event.SourceEndpointSensorId }}"
+          crowdstrike.account_creation_time_stamp: "{{ account_creation_time_stamp.datetime }}"
+          crowdstrike.activity_id: "{{ parsed_event.message.event.ActivityId }}"
+          crowdstrike.added_privilege: "{{ parsed_event.message.event.AddedPrivilege }}"
+          crowdstrike.attempt_outcome: "{{ parsed_event.message.event.AttemptOutcome }}"
+          crowdstrike.detect_description: "{{ parsed_event.message.event.DetectDescription }}"
+          crowdstrike.detect_name: "{{ parsed_event.message.event.DetectName }}"
+          crowdstrike.idp_policy_rule_trigger: "{{ parsed_event.message.event.IdpPolicyRuleTrigger }}"
+          crowdstrike.pattern_id: "{{ parsed_event.message.event.PatternId }}"
+          crowdstrike.previous_privileges: "{{ parsed_event.message.event.PreviousPrivileges }}"
+          destination.domain: "{{ parsed_event.message.event.TargetEndpointHostName }}"
+          event.end: "{{ parse_event_end_date.datetime }}"
+          event.reason: "{{ parsed_event.message.event.Objective }}"
+          event.reference: "{{ parsed_event.message.event.FalconHostLink }}"
+          event.severity: "{{ parsed_event.message.event.Severity }}"
+          event.start: "{{ parse_event_start_date.datetime }}"
+          event.url: "{{ parsed_event.message.event.FalconHostLink }}"
+          host.id: "{{ parsed_event.message.event.SourceEndpointAccountObjectGuid }}"
+          host.name: "{{ parsed_event.message.event.SourceEndpointHostName }}"
+          rule: "{{ parsed_event.message.event.IdpPolicyRuleAction }}"
```

Comment:
My bad. I made a mistake with the spec.
I suggest moving this field to `event.action`
```suggestion
          event.action: "{{ parsed_event.message.event.IdpPolicyRuleAction }}"
```